shell/src/main/java/alluxio/shell/command/CopyFromLocalCommand.java

```diff
@@ -91,6 +91,11 @@ private void copyFromLocalDir(File srcDir, AlluxioURI dstPath)
     createDstDir(dstPath);
     List<String> errorMessages = new ArrayList<>();
     File[] fileList = srcDir.listFiles();
+    if (fileList == null) {
+      String errMsg = String.format("copyFromLocal %s %s failed!",
+              srcDir.toString(), dstPath.toString());
+      throw new IOException(errMsg);
```

Comment:
Maybe something like `"Failed to list files for directory %s", srdDir` so that the user has a better idea where to look for the problem?

We should add the error message to the `errorMessages` list instead of throwing it so that the destination directory gets cleaned up.